packages/dhis2/src/Adaptor.js

```diff
@@ -5,14 +5,23 @@ import _ from 'lodash';
 const { indexOf } = _;
 import {
   CONTENT_TYPES,
-  generateUrl,
   handleResponse,
   prettyJson,
   selectId,
   shouldUseNewTracker,
   ensureArray,
+  generateUrlPath,
+  request,
 } from './Utils';
-import { request } from './Client';
+
+/**
+ * Options object
+ * @typedef {Object} RequestOptions
```

Comment:
This should include `apiVersion` as well, no? So that we can override the API version for a specific request.

Are there any other options we need to document? Stuff used by dhsi2 but not by the common helper?

Reply:
I have added it. We are already checking for it in the `prefixVersionToPath()` function